src/main/java/bio/terra/service/common/CommonFlightUtils.java

```diff
@@ -14,4 +21,27 @@ public static Instant getCreatedAt(FlightContext context) {
     }
     return null;
   }
+
```

Comment:
Specifically patterned after an allowList approach because the context contains some elements we don't want in Journal entries including tokens.  Since we know the "REQUEST" contains encoded JSON, I've added the hint to the key name so that we can detect this on the front end and and unwind it.